### PR TITLE
Add GeoJSON datatype

### PR DESCRIPTION
**Pull request by bgruening**
Needed by our ecology community.

### CODE REVIEW

**Comment by bgruening** [comment]
Mh, does the failing test mean that Galaxy is modifying the uploaded file?

**Comment by martenson** [comment]
@bgruening hmm, yes, the temp file has different md5 sum :/

File: lib/galaxy/datatypes/test/gis.geojson [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          ]
+        ]
+      },
+      "properties": {
+        "prop0": "value0",
+        "prop1": { "this": "that" }
+      }
+    }
+  ]
+}
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/test/gis.geojson]: It may be the missing end of line here.

Comment (martenson) [reply lib/galaxy/datatypes/test/gis.geojson]: correct, that is it

**Comment by bgruening** [comment]
Merci!

**Pull request merged**